src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacCard.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2020 Connor Holloway
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.isaac.dos;
+
+
+import uk.ac.cam.cl.dtg.segue.dos.content.Content;
+import uk.ac.cam.cl.dtg.segue.dos.content.DTOMapping;
+import uk.ac.cam.cl.dtg.segue.dos.content.Image;
+import uk.ac.cam.cl.dtg.segue.dos.content.JsonContentType;
+import uk.ac.cam.cl.dtg.isaac.dto.IsaacCardDTO;
+
+/**
+ * A card with optional picture and link.
+ */
+@DTOMapping(IsaacCardDTO.class)
+@JsonContentType("card")
```

Comment:
In the front-end branch, src/app/components/content/IsaacContent.tsx  is currently  looking for json objects with the property "type" with value "isaacCard". They need to be consistent for the front end to know which component to render. I think I prefer `card` as there is nothing particularly Isaac specific about it.